<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui</class>
 <widget class="QWidget" name="gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DTP</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Range_100</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Range_125</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Range_150</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="range_125">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>256</width>
     <height>151</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ins_50_1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_50_2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_50_3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_100_1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_100_2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_100_3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_200_1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_200_2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_200_3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_300_1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_300_2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_300_3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_400_1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_400_2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_400_3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_500_1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_500_2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_500_3</string>
    </property>
   </item>
  </widget>
  <widget class="QListWidget" name="range_150">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>50</y>
     <width>256</width>
     <height>151</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ins_50_1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_50_2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_50_3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_100_1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_100_2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_100_3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_200_1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_200_2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_200_3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_300_1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_300_2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_300_3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_400_1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_400_2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_400_3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_500_1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_500_2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_500_3</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>311</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Selection - krizanje (p_c) = </string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="gui_c">
        <property name="text">
         <string>0.8</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Selection - mutacija (p_m) =</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="gui_m">
        <property name="text">
         <string>0.8</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>pm = </string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="gui_pm">
        <property name="text">
         <string>0.4</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>380</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QPushButton" name="gui_ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="range_100">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>256</width>
     <height>151</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ins_050_1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_050_2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_050_3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_100_1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_100_2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_100_3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_200_1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_200_2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_200_3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_300_1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_300_2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_300_3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_400_1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_400_2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_400_3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_500_1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_500_2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ins_500_3</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>291</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Velicina populacije (pop) =</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="gui_pop_size">
        <property name="text">
         <string>200</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Broj generacija (n) =</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="gui_n">
        <property name="text">
         <string>1000</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Vjerojatnost krizanja (p) = </string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="gui_p">
        <property name="text">
         <string>0.8</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="gui_text">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>410</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>380</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QPushButton" name="gui_ok_2">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>410</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Rješenje: </string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
